screenshare/package-lock.json

```diff
@@ -1,8 +1,2732 @@
 {
   "name": "highfidelity_screenshare",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
+    "": {
+      "name": "highfidelity_screenshare",
+      "version": "1.0.0",
```

Comment:
I don't think Screenshare's package-lock.json file needs any changes in this PR?

Reply:
I have no idea how this got in there, I thought I reverted it before submitting.